AppCenter/AppCenter/Internals/Channel/MSChannelGroupDefault.m

```diff
@@ -236,7 +236,8 @@ - (void)setMaxStorageSize:(long)sizeInBytes completionHandler:(nullable void (^)
   });
 }
 
-- (void)authTokenContext:(__unused MSAuthTokenContext *)authTokenContext didReceiveAuthToken:(NSString *)authToken {
+- (void)authTokenContext:(__unused MSAuthTokenContext *)authTokenContext didReceiveAuthToken:(NSString *)authToken
+              forNewUser:(__unused BOOL)isNewUser {
```

Comment:
@clpolet @ElektrojungeMS @jaelim-ms What do you think about this method naming? The `forNewUser:` part sounds strange to me.

Reply:
"forNewUser" I think we are expecting a user object not really a Boolean.